modules/core/test/load-file/load-image.spec.js

```diff
@@ -10,7 +12,8 @@ const TEST_CASES = [
     url: `data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAIAAAACCAYAAA
 Bytg0kAAAAFElEQVQIW2P8z/D/PwMDAwMjjAEAQOwF/W1Dp54AAAAASUVORK5CYII=`,
     width: 2,
-    height: 2
+    height: 2,
+    worker: false
```

Comment:
`readFile` does not support data URLs

Reply:
@ibgreen should we support data URL to arraybuffer with `readFile`?

Reply:
Yes I think so, this was always the intention. 

For comparison, `fetch` supports data urls.

I am increasingly thinking that we should offer a `fetchFile` that returns a mock fetch response object under Node.js.

This would enable using the same `fetchFile` method with streaming and "atomic" loaders, and also would allow for progress tracking on streaming loader.

Hopefully this will also help to consolidate code, the loaders.gl core is way too big right now according to the `metrics` script.